sdk/nodejs/runtime/closure/codePaths.ts

```diff
@@ -87,6 +89,7 @@ export async function computeCodePaths(
 
     let options: CodePathOptions;
     if (Array.isArray(optionsOrExtraIncludePaths)) {
+        log.warn("'function computeCodePaths(string[])' is deprecated. Use the [computeCodePaths] overload that takes a [CodePathOptions] instead.");
```

Comment:
general idea.  Name the symbol and say it's deprecated (in this case, i need to include the `string[]` portion as this is an overload).  Then have a message saying what to do.  The message here matches the messae in the `@deprecated` message above.